app/javascript/react/components/SessionFilters/CrowdMapGridSize.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useCallback } from "react";
+
+import { setFetchingCrowdMapData } from "../../store/crowdMapSlice";
+import { useAppDispatch } from "../../store/hooks";
+import { UrlParamsTypes, useMapParams } from "../../utils/mapParamsHandler";
+import * as S from "./SessionFilters.style";
+
+const gridSizes = [1, 10, 20, 30, 40];
+export const defaultGridSize = 51 - gridSizes[2];
+
+const CrowdMapGridSize = () => {
```

Comment:
Something is wrong with grid size. The sizes are set differently than in production. Here are some screen shots with size 40 and 20:
![Screenshot 2024-08-27 at 09 55 53](https://github.com/user-attachments/assets/52df1e77-a41f-49e1-a7e4-b559a940b1cc)
![Screenshot 2024-08-27 at 09 56 00](https://github.com/user-attachments/assets/0fb00121-526b-4925-acf7-a5562461c817)
![Screenshot 2024-08-27 at 09 56 24](https://github.com/user-attachments/assets/37572c9c-3633-4401-9171-26db1ce1606a)
![Screenshot 2024-08-27 at 09 56 32](https://github.com/user-attachments/assets/51c6a850-ba4b-4125-98d9-fe87ab4a87c2)